src/components/molecules/Forms/PostWork/PostWork.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react'
+
+import { Input } from 'components/atoms/Input/Input'
+import { Form } from 'components/molecules/Form/Form'
+
+import './PostWork.scss'
+
+interface PostWorkProps {
+  readonly onSubmit: (event: any) => any
+  readonly disabledButton?: boolean
+  readonly serverErrors?: any
+  readonly email: string
+}
+
+interface PostWorkState {
+  readonly name: string
+  readonly author: string
+  readonly datePublished: string
+  readonly dateCreated: string
+  readonly content: string
+}
+
+export class PostWork extends React.Component <PostWorkProps, PostWorkState> {
+  readonly state = {
+    name: 'My First Work!',
+    author: this.props.email,
```

Comment:
Change this to 'Your Name'. We don't want people mistakenly recording their email addresses on Po.et.

Reply:
Ok. Should we just have Name, Author, Tags, and Content require users to input the data rather than having a default value? That way we don't have to worry about a bunch of works with 'Your Name', 'My First Work!', etc?

Reply:
Yeah, agreed. Leave them all blank, except the two date fields.